deployment/ansible/roles/model-my-watershed.geoprocessing/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Update CA Certificates
+  shell: "update-ca-certificates -f"
```

Comment:
Confirmed that this is an instance of `ca-certificates-java` bug [1396760](https://bugs.launchpad.net/ubuntu/+source/ca-certificates-java/+bug/983302). This is actually an issue that should be fixed in `azavea.java`; I opened azavea/ansible-java#26 to track the issue, but I'm going to fix it right now.